dqmgui.spec

```diff
@@ -8,7 +8,7 @@
 %define webdoc_files %{installroot}/%{pkgrel}/128/doc
 %define cvs cvs://:pserver:anonymous@cmscvs.cern.ch:2401/cvs_server/repositories/CMSSW?passwd=AA_:yZZ3e
 
-Source0: git+https://github.com/cms-DQM/dqmgui_prod.git?obj=index128/%realversion&export=Monitoring&output=/Monitoring.tar.gz
+Source0: git+https://github.com/cms-DQM/dqmgui_prod.git?obj=index128_cc7/%realversion&export=Monitoring&output=/Monitoring.tar.gz
```

Comment:
I added this change since the tag we need (`9.9.0`) is on branch `index128_cc7` instead of `index128`, but I'm not exactly sure that this is needed. Does this build procedure use the git tag directly, based on what's specified on the `spec` file's first line?

Reply:
@nothingface0 , yes if the source url has `obj=branch/tag`  then `cmsBuild` uses `tag` . In your case `Source0` has `obj=index128/%realversion` i.e. `%realversion` (specified in the first line) is treated as tag and `cmsBuild` will download sources using the tag. so no need to change the branch but is better to keep the correct branch in source url

Reply:
Thank you @smuzaffar, so I leave it as `index128_cc7`, which is where the source code actually is, just in case.